block/block.go

```diff
@@ -0,0 +1,661 @@
+package block
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/EnsurityTechnologies/logger"
+	"github.com/fxamacker/cbor"
+	didmodule "github.com/rubixchain/rubixgoplatform/did"
+	"github.com/rubixchain/rubixgoplatform/util"
+)
+
+// ----------TokenChain----------------------
+// {
+// 	 "1" : TokenType        : int
+// 	 "2" : TransactionType  : string
+// 	 "3" : TokenOwner       : string
+// 	 "4" : GenesisBlock     : GenesisBlock
+//   "5" : TransInfo        : TransInfo
+//   "6" : SmartContract    : []byte
+//   "7" : QuorumSignature  : []string
+//   "8" : PledgeDetails    : map[string][]PledgeDetail
+//   "9" : SmartContractData : string
+//
+// }
+
```

Comment:
The comment above indicates what each number in the tokenchain means. Right ? @KiranHRubix

Reply:
yes that is correct